test/util/server.go

```diff
@@ -188,6 +189,7 @@ func CreateNodeCerts(nodeArgs *start.NodeArgs) error {
 	}
 
 	createNodeConfig := admin.NewDefaultCreateNodeConfigOptions()
+	createNodeConfig.Output = ioutil.Discard
```

Comment:
Just don't have any use for it at this point, nor any way to pass it back somewhere useful.